src/runtime/durable.ts

```diff
@@ -0,0 +1,71 @@
+// This file is only imported when the runtime is `Deno`. See `durable_cf.ts` in the same directory.
+
+const ENCODER = new TextEncoder();
+const MEMORY_DURABLE_STORAGE: Map<string, {
+	durable: Durable;
+	storage: Map<string, any>;
+}> = new Map();
+
+export class Durable {
+	storage: DurableStorage;
+
+	constructor(public id: string) {
```

Comment:
If you don't want module authors to call the constructor by accident, make sure to make this a `private constructor`.